dozer-types/src/models/app_config.rs

```diff
@@ -36,15 +40,77 @@ pub struct Config {
     #[serde(default = "default_home_dir")]
     ///directory for all process; Default: ~/.dozer
     pub home_dir: String,
+
     #[serde(skip_serializing_if = "Option::is_none")]
     #[prost(message, tag = "9")]
     /// flags to enable/disable features
     pub flags: Option<Flags>,
+
+    /// Cache lmdb max map size
+    #[prost(uint64, optional, tag = "10")]
+    #[serde(
+        default = "default_cache_max_map_size",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub cache_max_map_size: Option<u64>,
+
+    /// Pipeline lmdb max map size
+    #[prost(uint64, optional, tag = "11")]
+    #[serde(
+        default = "default_pipeline_max_map_size",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub pipeline_max_map_size: Option<u64>,
```

Comment:
I can't find how this config is passed to the pipeline. Am I missing something?

Reply:
It's not not passed yet